src/pages/search/registration_search/RegistrationSearchBar.tsx

```diff
@@ -104,8 +103,8 @@ export const RegistrationSearchBar = ({ registrationQuery }: Pick<SearchPageProp
         processSearchParamProperties(values, ResultParam.Issn);
         processSearchParamProperties(values, ResultParam.Doi);
         processSearchParamProperties(values, ResultParam.Handle);
-        processSearchParamProperties(values, ResultParam.GrantId);
-        processSearchParamProperties(values, ResultParam.CourseCode);
+        processSearchParamProperties(values, ResultParam.FundingSource);
```

Comment:
skal den bort herfra også? da denne alltid vil være i konflikt med filter som ligger i fasettene til venstre 🤔 Mulig jeg misforstår noe her da, ikke helt triviell den søkeflyten dessverre..
teorien min er at den kan fjernes, å legges det til en ny param når APIet får støtte for søk på tilskudds-ID, men kan som sagt være jeg misforstår

Reply:
JEg holder på å lage eget filter for tilskuddsID (fundingIdentifier) i search-api nå. Oppdaterer denne pull requesten når det er på plass :)